Fix embedding extraction example in README (#241)
juanmc2005/diart: README.md

```python
import rx.operators as ops
import diart.operators as dops
from diart.sources import MicrophoneAudioSource, FileAudioSource
from diart.blocks import SpeakerSegmentation, OverlapAwareSpeakerEmbedding

segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation")
embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding")

source = MicrophoneAudioSource()
# To take input from file:
# source = FileAudioSource("<filename>", sample_rate=16000)

> 

Reviewer: ```suggestion

# Make sure the model has been trained with the same sample rate
print(source.sample_rate)

```

```python
import rx.operators as ops
import diart.operators as dops
from diart.sources import MicrophoneAudioSource, FileAudioSource
from diart.blocks import SpeakerSegmentation, OverlapAwareSpeakerEmbedding

segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation")
embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding")

source = MicrophoneAudioSource()
# To take input from file:
# source = FileAudioSource("<filename>", sample_rate=16000)

> # Make sure the model has been trained with the same sample rate

Reviewer: ```suggestion
# Make sure the models have been trained with this sample rate
```

```python
import rx.operators as ops
import diart.operators as dops

> from diart.sources import MicrophoneAudioSource #, FileAudioSource

Reviewer: ```suggestion
from diart.sources import MicrophoneAudioSource, FileAudioSource
```

from diart.blocks import SpeakerSegmentation, OverlapAwareSpeakerEmbedding

> segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation", use_hf_token="")
> embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding", use_hf_token="")

Reviewer: ```suggestion
segmentation = SpeakerSegmentation.from_pretrained("pyannote/segmentation")
embedding = OverlapAwareSpeakerEmbedding.from_pretrained("pyannote/embedding")
```

I prefer that the unused parameters are left to their default value when there is one. In this case, the default will look for the token from the huggingface cli login

> mic = MicrophoneAudioSource()
> # To take input from file:
> # mic = FileAudioSource("<filename>", sample_rate=16000)

Reviewer: ```suggestion
source = MicrophoneAudioSource()
# To take input from file:
# source = FileAudioSource("<filename>", sample_rate=16000)
```

Changing the name to `source` because a file is not a mic

stream = mic.stream.pipe(
    # Reformat stream to 5s duration and 500ms shift

>     dops.rearrange_audio_stream(sample_rate=16000),

Reviewer: ```suggestion
    dops.rearrange_audio_stream(sample_rate=source.sample_rate),
```

Here it's better to tie the value to the one from the audio source

    ops.map(lambda wav: (wav, segmentation(wav))),
    ops.starmap(embedding)

> ).subscribe(on_next=lambda emb: print(emb)) #emb.shape to display shape

Reviewer: ```suggestion
).subscribe(on_next=lambda emb: print(emb.shape))
```

I prefer to leave the shape here so the example output isn't polluted. Users will understand that removing `.shape` will print the actual embeddings

> mic.read()

Reviewer: ```suggestion
source.read()
```

Renaming for consistency with my own suggestion

```

Output:

```

> # Displaying embeds: 
> tensor([[[-0.0442, -0.0327, -0.0910,  ...,  0.0134,  0.0209,  0.0050],
>          [-0.0404, -0.0342, -0.0780,  ...,  0.0395,  0.0334, -0.0140],
>          [-0.0404, -0.0342, -0.0780,  ...,  0.0395,  0.0334, -0.0140]]])
> tensor([[[-0.0724,  0.0049, -0.0660,  ...,  0.0359,  0.0247, -0.0256],
>          [-0.0462, -0.0256, -0.0642,  ...,  0.0417,  0.0273, -0.0135],
>          [-0.0459, -0.0263, -0.0639,  ...,  0.0412,  0.0269, -0.0131]]])

Reviewer: ```suggestion
# Shape is (batch_size, num_speakers, embedding_dim)
torch.Size([1, 3, 512])
torch.Size([1, 3, 512])
torch.Size([1, 3, 512])
```

As mentioned before, I prefer to leave the shape here
